docs/getting_started/tutorial.md

```diff
@@ -329,30 +357,31 @@ In your `blog_index_page.html` file enter the following content:
 ```
 
 Most of this should be familiar from our previous work with the `home_page.html` template, but we'll explain `get_children` a bit later.
-If you are coming from Django, the `pageurl` tag is similar to Django's `url` tag, but takes a Wagtail Page object as an additional argument.
+If you have a Django background, then you can notice that the `pageurl` tag is similar to Django's `url` tag, but takes a Wagtail Page object as an additional argument.
 
-Now we can create our new page. Here is how we will create one from the Wagtail admin interface: 
+Now you can create a new page. Here is how you can create one from the Wagtail admin interface: 
 
-1.  In the Wagtail admin, go to Pages, then Home.
-2.  Add a child page to the Home page by clicking on the "Actions" icon and selecting the option "Add child page".
-3.  Choose "Blog index page" from the list of the page types.
-4.  Use "Our Blog" as your page title, make sure it has the slug "blog" on the Promote tab, and publish it.
+1. Go to `http://127.0.0.1:8000/admin` and sign in with your admin user details.
+2. In the Wagtail [Admin interface](https://guide.wagtail.org/en-latest/concepts/wagtail-interfaces/#admin-interface), go to Pages, then click Home.
+3. Add a child page to the Home page by clicking **...** at the top of the screen and selecting the option **Add child page**.
+4.  Choose **Blog index page** from the list of the page types.
+5.  Use "Our Blog" as your page title, make sure it has the slug "blog" on the Promote tab, and publish it.
```

Comment:
Prettier will autoformat these numbers, best to run on the MD files so that indents are aligned.

Reply:
Perhaps we should stop trying to follow Prettier formatting for docs, since we disabled it there on purpose? As long as the Markdown renders correctly, it seems to me we’d be better off not worrying about formatting if it can’t be done automatically.

If we do want things to be correctly formatted, then we might as well set up Prettier for Markdown.

Reply:
I do think it's important, it's a bit annoying with list indents but it comes with a lot of benefits for common typos I've seen such as line breaks after/before headings, malformed backticks for code snippets, double spaces or extra spaces around words and sentences. 

Additionally, it picks up on the rst double backticks for inline code formatting and malformed sub-nesting of lists that seem ok in the editor but didn't work when rendered.

Happy to give up on nice formatting as a goal though, I know it does add friction. Every time I've done a pass on the formatter (every few months) I've found some kind of bug though. 

Previous discussions with comments/links - https://github.com/wagtail/wagtail/discussions/8753

Reply:
I've raised a PR to enable Prettier - can get some feedback there. https://github.com/wagtail/wagtail/pull/10597

Reply:
> It picks up on the rst double backticks 

❤️